### ISSUE TITLE
Possible mixing up of question and answer prompt

### ISSUE THREAD

**Issue by Dryhb** [LAION-AI/Open-Assistant#1284]
I encountered a strange conversation on prompt n° 335c55a1-6951-482a-8a04-f78c0fc4b9d8 : the first question is about recycled paper and the assistant answer is about politics and elections with not a signel visible link between the two. I doubt the mess up comes from the assistant but maybe it's a user made reply as assistant and the user copy pasted this answer from another result I've seen (another prompt which had the same (or at least very similar) answer. Otherwise it could be a mix up between the distribution of the answer and the question. (was in french)
![error distribution prompt 335c55a1-6951-482a-8a04-f78c0fc4b9d8](https://user-images.githubusercontent.com/66953926/217145403-38f87009-cb5d-4992-aa33-37bbc066107d.PNG)

**Comment by Dryhb** [comment]
I've seen other examples too, sometimes the assistant reply is the start of a another reply for a totally different question but the reply is cut off in the middle of a word/sentence.

**Comment by fozziethebeat** [comment]
These kinds of mix ups are good examples where the downvoting will be useful information.  I'm going to close this as working as intended given the ability to downvote

**Comment by GuilleHoardings** [comment]
@fozziethebeat I don't think that this is working as intended: it points to a logic or database error with questions mixed up, so maybe information is corrupted. Also, I didn't see it in previous versions and I see it a lot in the last hours. I believe that this requires further investigation.

**Comment by fozziethebeat** [comment]
Let me tag this as a backend issue then.

**Comment by andreaskoepf** [comment]
Thanks a lot for reporting these issues, we will investigate this. 

Here are links to the corresponding trees:

Task 335c55a1-6951-482a-8a04-f78c0fc4b9d8:
https://open-assistant.io/messages/e9368baf-6d30-4546-a944-de46af7fde3f

335c55a1-6951-482a-8a04-f78c0fc4b9d8 : Q(recycled paper) R(election and french politics)
https://open-assistant.io/messages/e9368baf-6d30-4546-a944-de46af7fde3f

e1412dc1-e622-4813-a81f-23347f973974 : Q(Conversation between dolls) R(Ubuntu Concept)
https://open-assistant.io/messages/75d2de77-9f00-4576-a5ed-64f75fbd9c99

4d17bcbd-d36e-4620-8eb5-487168b58f49 : Q(sorting a data file in python) R(Animal crossing answer)
https://open-assistant.io/messages/d761245f-170d-42ba-a379-e64778197a8c

93c3bcf2-46fe-408d-b7d0-6fa6233d017e : Q(Animal crossing question) R(recycled paper)
https://open-assistant.io/messages/442fdad6-b5db-41e9-8034-6ee99578fb19

e90564f9-4d9e-4228-9082-0a0f9523d2f2 : Q(frequency of a space event) R(info about the translation of "stack" to french)
https://open-assistant.io/messages/5b64073e-e8f2-4111-87fc-389b43aa1de2

**Comment by Dryhb** [comment]
Ho perfect, I'll do a little translation : 
335c55a1-6951-482a-8a04-f78c0fc4b9d8 : Q(recycled paper) R(election and french politics)
https://open-assistant.io/messages/e9368baf-6d30-4546-a944-de46af7fde3f 
 ==> two first childrens are fine and on topic but the 3rd one is completely off topic (politics and electionsà

e1412dc1-e622-4813-a81f-23347f973974 : Q(Conversation between dolls) R(Ubuntu Concept)
https://open-assistant.io/messages/75d2de77-9f00-4576-a5ed-64f75fbd9c99
==>1st children is the wrong one (talking about ubuntu) and second is fine, on topic

4d17bcbd-d36e-4620-8eb5-487168b58f49 : Q(sorting a data file in python) R(Animal crossing answer)
https://open-assistant.io/messages/d761245f-170d-42ba-a379-e64778197a8c
==> all on topic except that one message (7 of the page, 2nd message of the second ancestor) about animal crossing

93c3bcf2-46fe-408d-b7d0-6fa6233d017e : Q(Animal crossing question) R(recycled paper)
https://open-assistant.io/messages/442fdad6-b5db-41e9-8034-6ee99578fb19
==> two first children on topic but 3rd one completely off (about recycled paper)

e90564f9-4d9e-4228-9082-0a0f9523d2f2 : Q(frequency of a space event) R(info about the translation of "stack" to french)
https://open-assistant.io/messages/5b64073e-e8f2-4111-87fc-389b43aa1de2
==> 5th message (first after the first ancestor) is the one off topic, I can also see the very last message (labeled as ancestor ???) is strange too as it's cut off and wrong language but that could be "normal" answer from the assistant.

**Comment by andreaskoepf** [comment]
Moderation team took care of this.
